demos/index.html

```diff
@@ -14,19 +13,63 @@
         <script type="text/javascript" src="../src/js/dataEntry.js"></script>
         <script type="text/javascript" src="../src/js/dataEntryPanel.js"></script>
         <script type="text/javascript" src="../src/js/chartAuthoring.js"></script>
+        <script type="text/javascript" src="../src/js/d3ViewComponent.js"></script>
+        <script type="text/javascript" src="../src/js/legend.js"></script>
+        <script type="text/javascript" src="../src/js/pie.js"></script>
+        <script type="text/javascript" src="../src/js/pieChart.js"></script>
+        <script type="text/javascript" src="../src/js/d3Utils.js"></script>
+
 
         <title>Chart Authoring Tool</title>
+        <style>
+            .floec-ca-pieChart-pie {
+                float: left;
+                width: 400px;
+            }
+            .floec-ca-pieChart-legend {
+                float: left;
+                width: 300px;
+            }
+            .floe-ca-pieChart-legend-table {
+                border: 1px solid black;
+            }
+            .floe-ca-pieChart-legend-table thead th {
+                border: 1px solid black;
+                padding: 2px;
+            }
+            .floe-ca-pieChart-legend-table-row {
+                border: 1px solid black;
+            }
+            .floe-ca-pieChart-legend-table-row td {
+                border: 1px solid black;
+                padding: 2px;
+            }
+            #floec-chartAuthoring-dataEntryPanel {
+                clear: both;
+            }
+        </style>
     </head>
 
     <body>
+        <div id="floec-chartAuthoring">
+        <div id="floec-chartAuthoring-pieChart"></div>
+        <div id="floec-chartAuthoring-dataEntryPanel"></div>
```

Comment:
Indentations are off at line 55, 56 as they are sub-elements of the div at line 54.